Remove old email alias (#4598)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

<ApiLifecycle access="ie" />

> > **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

<ApiLifecycle access="ie" />

> > **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](https://support.okta.com/) for information on the Identity Engine.
```

| providers | List of configured Identity Providers that a given Rule can route to | array     | Yes      |

> **Note:** Ability to define multiple providers is a part of the Identity Engine.

> > Please [contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

<ApiLifecycle access="ie" />

> > **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```

<ApiLifecycle access="ie" />

> > **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for further information.

Reviewer: ```suggestion
> **Note:** This feature is only available as a part of the Identity Engine. [Contact support](https://support.okta.com/) for information on the Identity Engine.
```
